src/form.rs

```diff
@@ -92,3 +110,8 @@ cfg_if::cfg_if! {
         }
     }
 }
+
+// Blanket implementation
+impl JsonSchemaMaybe for &'static str {}
+#[cfg(any(feature = "std", feature = "decode"))]
+impl JsonSchemaMaybe for crate::prelude::string::String {}
```

Comment:
True blanket impl would be `impl<T> JsonSchemaMaybe for T` for the feature not enabled and  `impl<T> JsonSchemaMaybe for T where T: JsonSchema` when the `schema` feature is enabled